src/Symfony/Bundle/SecurityBundle/Resources/config/security_acl.xml

```diff
@@ -13,13 +13,12 @@
         <parameter key="security.acl.object_identity_retrieval_strategy.class">Symfony\Component\Security\Acl\Domain\ObjectIdentityRetrievalStrategy</parameter>
         <parameter key="security.acl.security_identity_retrieval_strategy.class">Symfony\Component\Security\Acl\Domain\SecurityIdentityRetrievalStrategy</parameter>
 
-        <parameter key="security.acl.cache.doctrine.class">Symfony\Component\Security\Acl\Domain\DoctrineAclCache</parameter>
```

Comment:
Both `security_acl_dbal.xml` and `security_acl` are always loaded together at the same time.